src/Controllers/EditEntryController.php

```diff
@@ -8,22 +8,49 @@
 
 class EditEntryController extends Controller
 {
-    private $BlogModel;
+    private $blogModel;
 
     /**
      * editEntryController constructor.
-     * @param $BlogModel
+     * @param $blogModel
      */
-    public function __construct($BlogModel)
+    public function __construct($blogModel)
     {
-        $this->BlogModel = $BlogModel;
+        $this->blogModel = $blogModel;
     }
 
     public function __invoke(Request $request, Response $response, array $args)
     {
-        $EditEntry = $request->getParsedBody();
-        $DbResult = $this->BlogModel->editEntry($EditEntry);
-        return $response->withHeader('location', '/');
+        $responseData =
+            [
+                'success' => false,
+                'message' => '',
+                'data' => []
+            ];
+
+        $blogPost = new Valitron\Validator($_POST);
+        $blogPost->rule('required', ['GUID']);
+
+        if($blogPost->validate()) {
+
+            $result = $this->blogModel->EditEntry($blogPost);
+

```

Comment:
ok so here you want to edit the blog post right, but with what? All you are sending the EditEntry method is the GUID by the looks of it, not the data that should be edited.